app/views/handlers/list_add_question.py

```diff
@@ -37,7 +36,7 @@ def handle_post(self):
 
         return super().handle_post()
 
-    def _resolve_custom_page_title_vars(self) -> MutableMapping:
+    def _resolve_custom_page_title_vars(self):  # type: ignore
```

Comment:
How come this one was changed? No comment here for explanation?

Reply:
It was a linting error. Have updated after Kyles comment